macros/psng_manual_change.ngc

```diff
@@ -46,20 +49,21 @@ O400 endif
 
 F #<_hal[probe.ps_searchvel]>
 G91
-G38.2 Z #<_ini[TOOLSENSOR]MAXPROBE>
-G38.4 Z2
-G0 Z0.5
-; This is commented out only for sim.
+G38.2 Z#<_ini[TOOLSENSOR]MAXPROBE>
+G1 Z[#<_hal[probe.ps_probe_latch]>] F#<_ini[TOOLSENSOR]RAPID_SPEED>
 F #<_hal[probe.ps_probevel]>
-G38.2 Z-3
+G4 P0.5
+G38.2 Z[-#<_hal[probe.ps_probe_latch]>*2]
 
 O500 if [#5070 EQ 0]
 G90
 O500 return [-3] ; indicate probe contact failure to epilog
 O500 endif
 
+F#<_ini[TOOLSENSOR]RAPID_SPEED>
 G90
-G53 G0 Z[#<_ini[CHANGE_POSITION]Z>]
+G53 G1 Z[#<_ini[CHANGE_POSITION]Z>]
+G53 G1 X[#<_ini[CHANGE_POSITION]X>] Y[#<_ini[CHANGE_POSITION]Y>]
```

Comment:
Why move back to the change position? Given this isn't the same position as we were at when we entered the macro, there doesn't seem to be much value in moving back here.